<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceMakerClass</class>
 <widget class="QWidget" name="SourceMakerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>素材压缩工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SourceMaker.qrc">
    <normaloff>:/SourceMaker/Resources/Region_Icon.png</normaloff>:/SourceMaker/Resources/Region_Icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>请选择压缩或减压缩目录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_savePath">
         <property name="toolTip">
          <string>45</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>32766</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SavePath">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>添加多个要压缩的文件夹可实现批量压缩</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_press">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Press">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>压缩</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>编码方式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_CodeSytle">
         <property name="styleSheet">
          <string notr="true"> QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;

      border-left-width: 1px;
      border-left-color: darkgray;
      border-left-style: solid; 
      border-top-right-radius: 3px;
      border-bottom-right-radius: 3px;
  }
 QComboBox::down-arrow:on { 
      top: 1px;
      left: 1px;
  }</string>
         </property>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GB18030</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Big5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IBM850</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Windows-1250</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_unpress">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_UnPress">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>解压缩</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_MulitPress">
       <property name="toolTip">
        <string>将多个要压缩的文件夹放入同一个文件夹中</string>
       </property>
       <property name="text">
        <string>批量压缩</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200,200);
color: rgb(0, 20, 20);
font: 11pt &quot;Adobe 黑体 Std R&quot;;</string>
     </property>
     <property name="text">
      <string>特定编码压缩与解压缩</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SourceMaker.qrc"/>
 </resources>
 <connections/>
</ui>
